backend/app/tests.py

```diff
@@ -208,3 +212,163 @@ def test_add_document_to_corpus(self):
         corpus1.documents.add(doc2, doc3)
         self.assertEqual(list(corpus1.documents.all()), [doc1, doc2, doc3])
 
+
+class ProximityTestCase(TestCase):
+    """
+    Test Cases for the analysis functions in `proximity.py`
+    """
+
+    def setUp(self):
+        self.maxDiff = None
+
+        text_1 = "He went to get himself an ice cream, and he also got one for her. She was happy."
+        text_2 = "Fairest Cordelia, that art most rich being poor; Most choice, forsaken; and most loved her, despised, " \
+                 "herself and himself virtues here I hers hers seize upon. Lear then banishes his daughter to France."
+        text_3 = "She sells seashells by the seashore. He he reads books. She likes math. His father is scared of spiders. "
+
+        Document.objects.create_document(title='Text 1', year=2021, text=text_1)
+        Document.objects.create_document(title='Text 2', year=2021, text=text_2)
+        Document.objects.create_document(title='Text 3', year=2021, text=text_3)
+
+        corpus = Corpus.objects.create(
+            title='Test Corpus',
+            description='Testing the analysis functions in `proximity.py`'
+        )
+        corpus.documents.add(1, 2, 3)
+
+    def test_proximity(self):
+        male = Gender.objects.get(pk=1)
+        female = Gender.objects.get(pk=2)
```

Comment:
We may want to reference these by `label` rather than primary key, as there's a hypothetical where the primary key isn't consistent based on the seeds. I mean, that's true of the label as well, but it feels safer.

Reply:
Good point! I initially avoided referencing by `label` in the case that someone created similar constants and saved them to their local database (at least I have, so the label `Male` would return more than one object). I've referenced by `label` in addition to `pk` just to be safe. Since the newly created database for each `TestCase` should be empty except for the data in the migration files, the `.get()` lookup would only return a `Gender.DoesNotExist` error if we change migrations `0006` and `0009` (`0009` is in PR #48, but I've checked to make sure that it doesn't affect the `pk` ordering of any objects).